src/services/kpis/kpis.svc.coffee

```diff
@@ -122,8 +138,18 @@ angular
       promises = []
 
       createUrl = ImpacRoutes.kpis.alerts.create(kpi.id)
+
+
       for alert in alertsToCreate
-        promises.push $http.post(createUrl, {alert: _.pick(alert, ['service'])})
+        # Configure alert for inapp websockets with Pusher.
```

Comment:
I really think this should be part of the Alert model in mno-hub, upon creation of an 'inapp' alert...